tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -66,8 +66,14 @@
       </button>
       <button
         mat-menu-item
-        [disabled]="!seriesDataList || !seriesDataList.length"
+        [disabled]="!seriesDataList || !seriesDataList.length || isExtentAtDefault"
```

Comment:
Looking at the LineChart's impl, it seems like `isExtentAtDefault === !isViewBoxOverridden` always.  If so, would it be simpler to have the scalar card component's template just call `!this.lineChart?.getIsViewboxOverridden()`?

That way, we might need to worry about making sure the event emitter is properly called whenever `isViewBoxOverridden` changes.

Reply:
Done. I am a bit afraid of the dirty check being imperfect when you invoke a method but it seems negligible and not too terrible right now (ngAfterContentChecked or whatever gets invoked on every single mousemove event; the method is invoked on mouse move as well)